### ISSUE TITLE
EDB Upgrade Fix

### ISSUE THREAD

**Issue by caggles** [BCDevOps/developer-experience#3375]
**Describe the issue**
After some investigation, I found out what the issues with the EDB upgrade are - a breaking change in Openshift with 1.16. I've been working with the EDB support team and Jose now to actually perform the upgrade. 

**Definition of done**
- The EDB operator will be upgraded to 1.16

**Comment by caggles** [comment]
Currently blocked by an investigation by EDB's support team to ensure our upgrade path works properly with CCM.

**Comment by caggles** [comment]
This is now unblocked and is currently in progress. We have upgraded to version 1.15.5 and must wait for other teams to upgrade to a similar version. We will upgrade from there to 1.16, 1.17, and finally 1.18.

**Comment by caggles** [comment]
Unfortunately this is blocked again. Teams upgraded to 1.15.5 and we attempted the upgrade to 1.16, but another error is blocking the upgrade. Support led us to believe that going from 1.15.5 to 1.16 should be very smooth, so I've opened a new ticket to figure out what's going on here.

**Comment by caggles** [comment]
we did iiiiiiiiit, thanks @tmorik :D
